packages/react/src/date-range-picker/DateRangePicker.tsx

```diff
@@ -354,7 +354,15 @@ export const DateRangePicker = ({
 					{message && invalid ? (
 						<FieldMessage id={messageId}>{message}</FieldMessage>
 					) : null}
-					<Flex {...popover.getReferenceProps()} flexWrap="wrap" inline gap={1}>
+					<Flex
+						{...popover.getReferenceProps()}
+						flexWrap="wrap"
+						inline
+						gap={1}
+						css={{
+							paddingRight: toInvalid || fromInvalid ? undefined : '1.25rem',
```

Comment:
```suggestion
							paddingRight: invalid ? undefined : '1.25rem',
```

Reply:
I guess we'd need to use and combine the correct tokens from the error padding & border